Use openvpn-gui and interactive service to start VPN connections

lstipakov opened the pull request:
This enables more comprehensive testing of Windows client.

Configuration is now always stored in .ovpn file in $ENV:UserProfile\\OpenVPN\config directory.

When .\Start-LocalTest.ps1 is launched with -UseGUI 1:

 - connection profiles are refreshed in -gui by calling openvpn-gui.exe --command rescan

 - connection is started with openvpn-gui.exe --connect <config>

 - connection is stopped with openvpn-gui.exe --command disconnect <config>

Signed-off-by: Lev Stipakov <lev@openvpn.net>

flichtenheld reviewed (approved):
generally looks good to me. Haven't tested it.

lstipakov commented:
Thanks, suggestions applied. I also managed to commit "UseGUI 0" flag which I just set for testing. I reverted this PR, fixed the flag (so that AWS test will use gui/service) and merged changes in https://github.com/OpenVPN/openvpn-windows-test/pull/13.
